src/components/cost-management.svg

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Can we add this icon to chrome as well? In order to serve icons and images from one place?

Reply:
Yeah I'll add it there too